### DISCUSSION TITLE
Updating Forum Posts

### DISCUSSION THREAD

**Discussion by stephenwilliams** [disgoorg/disgo#277] [Help] [answered]
There is a way to create Forum Posts/Threads. Is there a way to update it post creation? Both the message and stuff like the tags?

**Comment by topi314** [comment, marked as answer]
sure, the post content is just a message with the same id as the thread, so just use the message update method in `rest`

a forum thread acts like a channel, so to update the tags use the update channel method in [`rest.UpdateChannel`](https://pkg.go.dev/github.com/disgoorg/disgo@v0.16.6/rest#Channels) with [`discord.GuildForumChannelUpdate`](https://pkg.go.dev/github.com/disgoorg/disgo@v0.16.6/discord#GuildForumChannelUpdate)

**Reply by stephenwilliams** [reply]
Thank you!

Also taking the time to say thanks for the library. This is a breath of fresh air compared to one of the other libraries I had tried!
